extensions/blocks/transcription/transcription.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Transcription Block.
+ *
+ * @since 9.x
+ *
+ * @package Jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\Transcription;
+
+use Automattic\Jetpack\Blocks;
+use Jetpack_Gutenberg;
+
+const FEATURE_NAME = 'transcription';
+const BLOCK_NAME   = 'jetpack/' . FEATURE_NAME;
+
+/**
+ * Registers the block for use in Gutenberg
+ * This is done via an action so that we can disable
+ * registration if we need to.
+ */
+function register_block() {
+	$deprecated = function_exists( 'gutenberg_get_post_from_context' );
+	$provides   = $deprecated ? 'providesContext' : 'provides_context';
+
+	Blocks::jetpack_register_block(
```

Comment:
Is there a reason we don't use this function for the dialogue block?

Reply:
yes! it's me. Updating...